### ISSUE TITLE
Your site needs a route with route type service-not-available

### ISSUE THREAD

**Issue by rajavelrntbci** [trailheadapps/az-insurance#12]
While installing the code , I was getting the below error - **[   In field: siteAdmin - no User named your.user@lwr.demo foundError   ]** .  i have fixed this by adding 'My UserName'

Error  force-app\main\default\sites\LWR_Demo_Marketing.site-meta.xml                                In field: siteAdmin - no User named your.user@lwr.demo foundError  force-app\main\default\navigationMenus\Marketing_Default_Navigation.navigationMenu-meta.xml  In field: Network - no Network named LWR Demo Marketing found

still not able to deploy : getting new issue.

Error  force-app\main\default\experiences\LWR_Demo_Marketing1.site-meta.xml  Your site needs a route with route type service-not-available. Update your file and try again.
ERROR running force:source:deploy:  Deploy failed.

Could you please check

**Comment by muenzpraeger** [comment]
@rajavelrntbci What type of org did you deploy the metadata to?

**Comment by rajavelrntbci** [comment]
> @rajavelrntbci What type of org did you deploy the metadata to?

Developer org .

**Comment by muenzpraeger** [comment]
Yup, that will need different steps. Note, that we provide at the moment only a setup for [scratch orgs](https://github.com/trailheadapps/az-insurance#installing-az-insurance-using-a-scratch-org).

**Comment by rajavelrntbci** [comment]
> Yup, that will need different steps. Note, that we provide at the moment only a setup for [scratch orgs](https://github.com/trailheadapps/az-insurance#installing-az-insurance-using-a-scratch-org).

Ok, let me practice in scratch org

**Comment by rajavelrntbci** [comment]
Hi ,
I tried in scratch org. still the same issue. please check the attached file.

![image](https://user-images.githubusercontent.com/11882840/125026900-1023ba00-e0a3-11eb-963f-f7b6c9335f0a.png)

**Comment by muenzpraeger** [comment]
Weird. I just ran the commands from the README, see output below.

```zsh
az-insurance on  main via ⬢ v14.15.0
➜ sfdx force:org:create -s -f config/project-scratch-def.json -a az-insurance-issue-12
Successfully created scratch org: 00D21000000K97KEAS, username: test-8gpkjylnck34@example.com

az-insurance on  main via ⬢ v14.15.0 took 1m1s with org az-insurance-issue-12
➜ sfdx force:community:create --name "Dummy" --templatename "Aloha" -p "dummy"

=== Create Site Result
NAME   MESSAGE                      ACTION
─────  ───────────────────────────  ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Dummy  Your Site is being created.  We’re creating your site. Run sfdx force:org:open -p _ui/networks/setup/SetupNetworksPage to view a list of your sites, and to confirm when this site is ready.

az-insurance on  main via ⬢ v14.15.0 took 11s with org az-insurance-issue-12
➜ sfdx force:source:deploy -m ApexClass,Layout,CustomObject,LightningComponentBundle,ManagedContentType,CustomObject,StaticResource,CustomTab,PermissionSet,Flow

*** Deploying with SOAP ***
Job ID | 0Af21000015XpmpCAC
SOURCE PROGRESS | ████████████████████████████████████████ | 49/49 Components
=== Deployed Source
FULL NAME                                                 TYPE                      PROJECT PATH
────────────────────────────────────────────────────────  ────────────────────────  ─────────────────────────────────────────────────────────────────────────────────────────────────────
ManagedContentController                                  ApexClass                 force-app/main/default/classes/ManagedContentController.cls
ManagedContentController                                  ApexClass                 force-app/main/default/classes/ManagedContentController.cls-meta.xml
NavigationLinkSetPickList                                 ApexClass                 force-app/main/default/classes/NavigationLinkSetPickList.cls
NavigationLinkSetPickList                                 ApexClass                 force-app/main/default/classes/NavigationLinkSetPickList.cls-meta.xml
NavigationMenuItemsController                             ApexClass                 force-app/main/default/classes/NavigationMenuItemsController.cls
NavigationMenuItemsController                             ApexClass                 force-app/main/default/classes/NavigationMenuItemsController.cls-meta.xml
LWR_Demo_Lead__c                                          CustomObject              force-app/main/default/objects/LWR_Demo_Lead__c/LWR_Demo_Lead__c.object-meta.xml
LWR_Demo_Lead__c.LWR_Demo_Layout                          CompactLayout             force-app/main/default/objects/LWR_Demo_Lead__c/compactLayouts/LWR_Demo_Layout.compactLayout-meta.xml
LWR_Demo_Lead__c.Address__c                               CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/Address__c.field-meta.xml
LWR_Demo_Lead__c.Email__c                                 CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/Email__c.field-meta.xml
LWR_Demo_Lead__c.First_Name__c                            CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/First_Name__c.field-meta.xml
LWR_Demo_Lead__c.Last_Name__c                             CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/Last_Name__c.field-meta.xml
LWR_Demo_Lead__c.Quote_Type__c                            CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/Quote_Type__c.field-meta.xml
LWR_Demo_Lead__c.Status__c                                CustomField               force-app/main/default/objects/LWR_Demo_Lead__c/fields/Status__c.field-meta.xml
LWR_Demo_Lead__c.All                                      ListView                  force-app/main/default/objects/LWR_Demo_Lead__c/listViews/All.listView-meta.xml
cardsSectionWithHeader/cardsSectionWithHeader.css         LightningComponentBundle  force-app/main/default/lwc/cardsSectionWithHeader/cardsSectionWithHeader.css
cardsSectionWithHeader/cardsSectionWithHeader.html        LightningComponentBundle  force-app/main/default/lwc/cardsSectionWithHeader/cardsSectionWithHeader.html
cardsSectionWithHeader/cardsSectionWithHeader.js          LightningComponentBundle  force-app/main/default/lwc/cardsSectionWithHeader/cardsSectionWithHeader.js
cardsSectionWithHeader/cardsSectionWithHeader.js          LightningComponentBundle  force-app/main/default/lwc/cardsSectionWithHeader/cardsSectionWithHeader.js-meta.xml
cmsBlogList/cmsBlogList.css                               LightningComponentBundle  force-app/main/default/lwc/cmsBlogList/cmsBlogList.css
cmsBlogList/cmsBlogList.html                              LightningComponentBundle  force-app/main/default/lwc/cmsBlogList/cmsBlogList.html
cmsBlogList/cmsBlogList.js                                LightningComponentBundle  force-app/main/default/lwc/cmsBlogList/cmsBlogList.js
cmsBlogList/cmsBlogList.js                                LightningComponentBundle  force-app/main/default/lwc/cmsBlogList/cmsBlogList.js-meta.xml
cmsBlogPost/cmsBlogPost.html                              LightningComponentBundle  force-app/main/default/lwc/cmsBlogPost/cmsBlogPost.html
cmsBlogPost/cmsBlogPost.js                                LightningComponentBundle  force-app/main/default/lwc/cmsBlogPost/cmsBlogPost.js
cmsBlogPost/cmsBlogPost.js                                LightningComponentBundle  force-app/main/default/lwc/cmsBlogPost/cmsBlogPost.js-meta.xml
cmsCustomizibleSection/cmsCustomizibleSection.css         LightningComponentBundle  force-app/main/default/lwc/cmsCustomizibleSection/cmsCustomizibleSection.css
cmsCustomizibleSection/cmsCustomizibleSection.html        LightningComponentBundle  force-app/main/default/lwc/cmsCustomizibleSection/cmsCustomizibleSection.html
cmsCustomizibleSection/cmsCustomizibleSection.js          LightningComponentBundle  force-app/main/default/lwc/cmsCustomizibleSection/cmsCustomizibleSection.js
cmsCustomizibleSection/cmsCustomizibleSection.js          LightningComponentBundle  force-app/main/default/lwc/cmsCustomizibleSection/cmsCustomizibleSection.js-meta.xml
cmsImageWithTitle/cmsImageWithTitle.css                   LightningComponentBundle  force-app/main/default/lwc/cmsImageWithTitle/cmsImageWithTitle.css
cmsImageWithTitle/cmsImageWithTitle.html                  LightningComponentBundle  force-app/main/default/lwc/cmsImageWithTitle/cmsImageWithTitle.html
cmsImageWithTitle/cmsImageWithTitle.js                    LightningComponentBundle  force-app/main/default/lwc/cmsImageWithTitle/cmsImageWithTitle.js
cmsImageWithTitle/cmsImageWithTitle.js                    LightningComponentBundle  force-app/main/default/lwc/cmsImageWithTitle/cmsImageWithTitle.js-meta.xml
cmsNewsCardWithButton/cmsNewsCardWithButton.css           LightningComponentBundle  force-app/main/default/lwc/cmsNewsCardWithButton/cmsNewsCardWithButton.css
cmsNewsCardWithButton/cmsNewsCardWithButton.html          LightningComponentBundle  force-app/main/default/lwc/cmsNewsCardWithButton/cmsNewsCardWithButton.html
cmsNewsCardWithButton/cmsNewsCardWithButton.js            LightningComponentBundle  force-app/main/default/lwc/cmsNewsCardWithButton/cmsNewsCardWithButton.js
cmsNewsCardWithButton/cmsNewsCardWithButton.js            LightningComponentBundle  force-app/main/default/lwc/cmsNewsCardWithButton/cmsNewsCardWithButton.js-meta.xml
cmsSingleDisplay/cmsSingleDisplay.html                    LightningComponentBundle  force-app/main/default/lwc/cmsSingleDisplay/cmsSingleDisplay.html
cmsSingleDisplay/cmsSingleDisplay.js                      LightningComponentBundle  force-app/main/default/lwc/cmsSingleDisplay/cmsSingleDisplay.js
cmsSingleDisplay/cmsSingleDisplay.js                      LightningComponentBundle  force-app/main/default/lwc/cmsSingleDisplay/cmsSingleDisplay.js-meta.xml
customThemeLayout/customThemeLayout.css                   LightningComponentBundle  force-app/main/default/lwc/customThemeLayout/customThemeLayout.css
customThemeLayout/customThemeLayout.html                  LightningComponentBundle  force-app/main/default/lwc/customThemeLayout/customThemeLayout.html
customThemeLayout/customThemeLayout.js                    LightningComponentBundle  force-app/main/default/lwc/customThemeLayout/customThemeLayout.js
customThemeLayout/customThemeLayout.js                    LightningComponentBundle  force-app/main/default/lwc/customThemeLayout/customThemeLayout.js-meta.xml
errorPanel/errorPanel.js                                  LightningComponentBundle  force-app/main/default/lwc/errorPanel/errorPanel.js
errorPanel/errorPanel.js                                  LightningComponentBundle  force-app/main/default/lwc/errorPanel/errorPanel.js-meta.xml
errorPanel/templates/inlineMessage.html                   LightningComponentBundle  force-app/main/default/lwc/errorPanel/templates/inlineMessage.html
errorPanel/templates/noDataIllustration.html              LightningComponentBundle  force-app/main/default/lwc/errorPanel/templates/noDataIllustration.html
footer/footer.css                                         LightningComponentBundle  force-app/main/default/lwc/footer/footer.css
footer/footer.html                                        LightningComponentBundle  force-app/main/default/lwc/footer/footer.html
footer/footer.js                                          LightningComponentBundle  force-app/main/default/lwc/footer/footer.js
footer/footer.js                                          LightningComponentBundle  force-app/main/default/lwc/footer/footer.js-meta.xml
footerList/footerList.html                                LightningComponentBundle  force-app/main/default/lwc/footerList/footerList.html
footerList/footerList.js                                  LightningComponentBundle  force-app/main/default/lwc/footerList/footerList.js
footerList/footerList.js                                  LightningComponentBundle  force-app/main/default/lwc/footerList/footerList.js-meta.xml
footerMenuItem/footerMenuItem.css                         LightningComponentBundle  force-app/main/default/lwc/footerMenuItem/footerMenuItem.css
footerMenuItem/footerMenuItem.html                        LightningComponentBundle  force-app/main/default/lwc/footerMenuItem/footerMenuItem.html
footerMenuItem/footerMenuItem.js                          LightningComponentBundle  force-app/main/default/lwc/footerMenuItem/footerMenuItem.js
footerMenuItem/footerMenuItem.js                          LightningComponentBundle  force-app/main/default/lwc/footerMenuItem/footerMenuItem.js-meta.xml
ldsUtils/ldsUtils.js                                      LightningComponentBundle  force-app/main/default/lwc/ldsUtils/ldsUtils.js
ldsUtils/ldsUtils.js                                      LightningComponentBundle  force-app/main/default/lwc/ldsUtils/ldsUtils.js-meta.xml
lightningNavigationButton/lightningNavigationButton.css   LightningComponentBundle  force-app/main/default/lwc/lightningNavigationButton/lightningNavigationButton.css
lightningNavigationButton/lightningNavigationButton.html  LightningComponentBundle  force-app/main/default/lwc/lightningNavigationButton/lightningNavigationButton.html
lightningNavigationButton/lightningNavigationButton.js    LightningComponentBundle  force-app/main/default/lwc/lightningNavigationButton/lightningNavigationButton.js
lightningNavigationButton/lightningNavigationButton.js    LightningComponentBundle  force-app/main/default/lwc/lightningNavigationButton/lightningNavigationButton.js-meta.xml
lightningRecordForm/lightningRecordForm.css               LightningComponentBundle  force-app/main/default/lwc/lightningRecordForm/lightningRecordForm.css
lightningRecordForm/lightningRecordForm.html              LightningComponentBundle  force-app/main/default/lwc/lightningRecordForm/lightningRecordForm.html
lightningRecordForm/lightningRecordForm.js                LightningComponentBundle  force-app/main/default/lwc/lightningRecordForm/lightningRecordForm.js
lightningRecordForm/lightningRecordForm.js                LightningComponentBundle  force-app/main/default/lwc/lightningRecordForm/lightningRecordForm.js-meta.xml
navigationMenu/navigationMenu.css                         LightningComponentBundle  force-app/main/default/lwc/navigationMenu/navigationMenu.css
navigationMenu/navigationMenu.html                        LightningComponentBundle  force-app/main/default/lwc/navigationMenu/navigationMenu.html
navigationMenu/navigationMenu.js                          LightningComponentBundle  force-app/main/default/lwc/navigationMenu/navigationMenu.js
navigationMenu/navigationMenu.js                          LightningComponentBundle  force-app/main/default/lwc/navigationMenu/navigationMenu.js-meta.xml
navigationMenuItem/navigationMenuItem.css                 LightningComponentBundle  force-app/main/default/lwc/navigationMenuItem/navigationMenuItem.css
navigationMenuItem/navigationMenuItem.html                LightningComponentBundle  force-app/main/default/lwc/navigationMenuItem/navigationMenuItem.html
navigationMenuItem/navigationMenuItem.js                  LightningComponentBundle  force-app/main/default/lwc/navigationMenuItem/navigationMenuItem.js
navigationMenuItem/navigationMenuItem.js                  LightningComponentBundle  force-app/main/default/lwc/navigationMenuItem/navigationMenuItem.js-meta.xml
navigationMenuLogo/navigationMenuLogo.html                LightningComponentBundle  force-app/main/default/lwc/navigationMenuLogo/navigationMenuLogo.html
navigationMenuLogo/navigationMenuLogo.js                  LightningComponentBundle  force-app/main/default/lwc/navigationMenuLogo/navigationMenuLogo.js
navigationMenuLogo/navigationMenuLogo.js                  LightningComponentBundle  force-app/main/default/lwc/navigationMenuLogo/navigationMenuLogo.js-meta.xml
robotoFont/robotoFont.css                                 LightningComponentBundle  force-app/main/default/lwc/robotoFont/robotoFont.css
robotoFont/robotoFont.js                                  LightningComponentBundle  force-app/main/default/lwc/robotoFont/robotoFont.js-meta.xml
sldsCarouselCmsPanel/sldsCarouselCmsPanel.html            LightningComponentBundle  force-app/main/default/lwc/sldsCarouselCmsPanel/sldsCarouselCmsPanel.html
sldsCarouselCmsPanel/sldsCarouselCmsPanel.js              LightningComponentBundle  force-app/main/default/lwc/sldsCarouselCmsPanel/sldsCarouselCmsPanel.js
sldsCarouselCmsPanel/sldsCarouselCmsPanel.js              LightningComponentBundle  force-app/main/default/lwc/sldsCarouselCmsPanel/sldsCarouselCmsPanel.js-meta.xml
sldsHeroCarousel/sldsHeroCarousel.css                     LightningComponentBundle  force-app/main/default/lwc/sldsHeroCarousel/sldsHeroCarousel.css
sldsHeroCarousel/sldsHeroCarousel.html                    LightningComponentBundle  force-app/main/default/lwc/sldsHeroCarousel/sldsHeroCarousel.html
sldsHeroCarousel/sldsHeroCarousel.js                      LightningComponentBundle  force-app/main/default/lwc/sldsHeroCarousel/sldsHeroCarousel.js
sldsHeroCarousel/sldsHeroCarousel.js                      LightningComponentBundle  force-app/main/default/lwc/sldsHeroCarousel/sldsHeroCarousel.js-meta.xml
cms_actionimage                                           ManagedContentType        force-app/main/default/managedContentTypes/cms_actionimage.managedContentType-meta.xml
cms_blogpost                                              ManagedContentType        force-app/main/default/managedContentTypes/cms_blogpost.managedContentType-meta.xml
cms_carouselimage                                         ManagedContentType        force-app/main/default/managedContentTypes/cms_carouselimage.managedContentType-meta.xml
MarketingExperienceStyles                                 StaticResource            force-app/main/default/staticresources/MarketingExperienceStyles.css
MarketingExperienceStyles                                 StaticResource            force-app/main/default/staticresources/MarketingExperienceStyles.resource-meta.xml
agentHamburgerMenuLogo                                    StaticResource            force-app/main/default/staticresources/agentHamburgerMenuLogo.png
agentHamburgerMenuLogo                                    StaticResource            force-app/main/default/staticresources/agentHamburgerMenuLogo.resource-meta.xml
azinsurancelogo                                           StaticResource            force-app/main/default/staticresources/azinsurancelogo.png
azinsurancelogo                                           StaticResource            force-app/main/default/staticresources/azinsurancelogo.resource-meta.xml
customerHamburgerMenuLogo                                 StaticResource            force-app/main/default/staticresources/customerHamburgerMenuLogo.png
customerHamburgerMenuLogo                                 StaticResource            force-app/main/default/staticresources/customerHamburgerMenuLogo.resource-meta.xml
marketingHamburgerIcon                                    StaticResource            force-app/main/default/staticresources/marketingHamburgerIcon.png
marketingHamburgerIcon                                    StaticResource            force-app/main/default/staticresources/marketingHamburgerIcon.resource-meta.xml
marketingHamburgerMenuLogo                                StaticResource            force-app/main/default/staticresources/marketingHamburgerMenuLogo.png
marketingHamburgerMenuLogo                                StaticResource            force-app/main/default/staticresources/marketingHamburgerMenuLogo.resource-meta.xml
marketingXIcon                                            StaticResource            force-app/main/default/staticresources/marketingXIcon.png
marketingXIcon                                            StaticResource            force-app/main/default/staticresources/marketingXIcon.resource-meta.xml
menuLogoAgent                                             StaticResource            force-app/main/default/staticresources/menuLogoAgent.png
menuLogoAgent                                             StaticResource            force-app/main/default/staticresources/menuLogoAgent.resource-meta.xml
menuLogoCustomer                                          StaticResource            force-app/main/default/staticresources/menuLogoCustomer.png
menuLogoCustomer                                          StaticResource            force-app/main/default/staticresources/menuLogoCustomer.resource-meta.xml
sldsBrandedCarousel                                       StaticResource            force-app/main/default/staticresources/sldsBrandedCarousel.css
sldsBrandedCarousel                                       StaticResource            force-app/main/default/staticresources/sldsBrandedCarousel.resource-meta.xml
LWR_Demo_Lead__c                                          CustomTab                 force-app/main/default/tabs/LWR_Demo_Lead__c.tab-meta.xml
LWR_Marketing_Builder                                     PermissionSet             force-app/main/default/permissionsets/LWR_Marketing_Builder.permissionset-meta.xml
New_LWR_Demo_Lead                                         Flow                      force-app/main/default/flows/New_LWR_Demo_Lead.flow-meta.xml

az-insurance on  main via ⬢ v14.15.0 took 37s with org az-insurance-issue-12
➜ sfdx force:source:deploy -m ApexPage,CustomSite,ExperienceBundle,NavigationMenu,Network,Profile

*** Deploying with SOAP ***
Job ID | 0Af21000015Xr5MCAS
SOURCE PROGRESS | ████████████████████████████████████████ | 6/6 Components
=== Deployed Source
FULL NAME                                              TYPE              PROJECT PATH
─────────────────────────────────────────────────────  ────────────────  ───────────────────────────────────────────────────────────────────────────────────────────
CommunitiesLanding                                     ApexPage          force-app/main/default/pages/CommunitiesLanding.page
CommunitiesLanding                                     ApexPage          force-app/main/default/pages/CommunitiesLanding.page-meta.xml
LWR_Demo_Marketing                                     CustomSite        force-app/main/default/sites/LWR_Demo_Marketing.site-meta.xml
LWR_Demo_Marketing1/brandingSets/buildYourOwnLWR.json  ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/brandingSets/buildYourOwnLWR.json
LWR_Demo_Marketing1/config/lWRDemoMarketing.json       ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/config/lWRDemoMarketing.json
LWR_Demo_Marketing1/config/languages.json              ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/config/languages.json
LWR_Demo_Marketing1/config/mainAppPage.json            ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/config/mainAppPage.json
LWR_Demo_Marketing1/routes/blog.json                   ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/blog.json
LWR_Demo_Marketing1/routes/blogPost.json               ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/blogPost.json
LWR_Demo_Marketing1/routes/error.json                  ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/error.json
LWR_Demo_Marketing1/routes/getAQuote.json              ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/getAQuote.json
LWR_Demo_Marketing1/routes/home.json                   ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/home.json
LWR_Demo_Marketing1/routes/quoteConfirmation.json      ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/routes/quoteConfirmation.json
LWR_Demo_Marketing1/themes/buildYourOwnLWR.json        ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/themes/buildYourOwnLWR.json
LWR_Demo_Marketing1/views/blog.json                    ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/blog.json
LWR_Demo_Marketing1/views/blogDetail.json              ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/blogDetail.json
LWR_Demo_Marketing1/views/error.json                   ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/error.json
LWR_Demo_Marketing1/views/getAQuote.json               ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/getAQuote.json
LWR_Demo_Marketing1/views/home.json                    ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/home.json
LWR_Demo_Marketing1/views/quoteConfirmation.json       ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1/views/quoteConfirmation.json
default/experiences/LWR_Demo_Marketing1.site           ExperienceBundle  force-app/main/default/experiences/LWR_Demo_Marketing1.site-meta.xml
Marketing_Default_Navigation                           NavigationMenu    force-app/main/default/navigationMenus/Marketing_Default_Navigation.navigationMenu-meta.xml
LWR Demo Marketing                                     Network           force-app/main/default/networks/LWR Demo Marketing.network-meta.xml
LWR Demo Marketing Profile                             Profile           force-app/main/default/profiles/LWR Demo Marketing Profile.profile-meta.xml

az-insurance on  main via ⬢ v14.15.0 took 33s with org az-insurance-issue-12
➜
```

**Comment by muenzpraeger** [comment]
@rajavelrntbci Can you please run from your `D:\LWRAZ` directory the following command, and share the output here?

`sfdx force:org:display`

Please redact data like the `Access Token`, and your PII data (i. e. email address).

**Comment by rajavelrntbci** [comment]
> sfdx force:org:display

![image](https://user-images.githubusercontent.com/11882840/125081788-079fa380-e0e4-11eb-8246-15b8718871c9.png)

**Comment by jorgesolebur** [comment]
Hi there @muenzpraeger  ! I am having same issue as @rajavelrntbci :
![image](https://user-images.githubusercontent.com/31279773/125147910-445db000-e161-11eb-8fd4-4d92243afefa.png)

The scratch org has been created from a developer edition dev hub, but I believe that should not be an issue because we are using the same scratch org definition file

**Comment by jorgesolebur** [comment]
> Hi there @muenzpraeger ! I am having same issue as @rajavelrntbci :
> ![image](https://user-images.githubusercontent.com/31279773/125147910-445db000-e161-11eb-8fd4-4d92243afefa.png)
> 
> The scratch org has been created from a developer edition dev hub, but I believe that should not be an issue because we are using the same scratch org definition file

Using the last version of CLI: sfdx-cli/7.109.0 darwin-x64 node-v14.17.1

**Comment by rajavelrntbci** [comment]
> > Hi there @muenzpraeger ! I am having same issue as @rajavelrntbci :
> > ![image](https://user-images.githubusercontent.com/31279773/125147910-445db000-e161-11eb-8fd4-4d92243afefa.png)
> > The scratch org has been created from a developer edition dev hub, but I believe that should not be an issue because we are using the same scratch org definition file
> 
> Using the last version of CLI: sfdx-cli/7.109.0 darwin-x64 node-v14.17.1

Strange issue for me also...

**Comment by muenzpraeger** [comment]
Yup, super strange. Still trying to verify why that happens to y'all. As you can see [in the CI/CD logs here](https://github.com/trailheadapps/az-insurance/pull/13/checks?check_run_id=3044497837) the metadata deploys there without any issues (and it's not a special devhub).

**Comment by kdinluks** [comment]
@rajavelrntbci if you're using scratch org, please make sure to add PublishExpBuilderBasedSna and PublishAuraExpBuilderBasedSna to your scratch org configuration.

**Comment by jorgesolebur** [comment]
> @rajavelrntbci if you're using scratch org, please make sure to add PublishExpBuilderBasedSna and PublishAuraExpBuilderBasedSna features to your scratch org configuration.

@kdinluks I added them :
![image](https://user-images.githubusercontent.com/31279773/125382476-4f664980-e3c8-11eb-907d-d20b107c9e67.png)
but still same issue: 
![image](https://user-images.githubusercontent.com/31279773/125382525-61e08300-e3c8-11eb-9856-5f5018c14bbd.png)

@muenzpraeger which script are you using to create teh scratch org? I am using bin/install-scratch.sh

**Comment by msnimbal** [comment]
Same issue. Looks like we need a route for Service Not Available in experiences/routes folder but not sure what is the configuration of that json or we need to update the LWR_Demo_Marketing1.site-meta.xml? or is because scratch org is older version?

```
Error  force-app/main/default/experiences/LWR_Demo_Marketing1.site-meta.xml  Your site needs a route with route type service-not-available. Update your file and try again.
ERROR running force:source:deploy:  Deploy failed.
```

**Comment by collymitch** [comment]
Same error - I was able to overcome by copying (and editing) a servicenotavailable route and view from a LWR community I manually created from setup - had to change the appPageId's to match the id from the mainAppPage config file.  Funnily enough I was unable to deploy that GUI built community as a favicon reference was missing from the source retrieved from salesforce so not everything working perfect just yet.

**Comment by muenzpraeger** [comment]
I just merged #14 into `main`, which fixed the issue. There seems to be a glitch (in some DE org types) that lead to the above message. It's investigated.

**Comment by rajavelrntbci** [comment]
> I just merged #14 into `main`, which fixed the issue. There seems to be a glitch (in some DE org types) that lead to the above message. It's investigated.

Perfect, working now. Thanks
